articles/data-lake-vs-data-warehouse-vs-database/index.md

```diff
@@ -0,0 +1,88 @@
+
+- **Public sector**
+
+The public sector relies on data warehouses for intelligence gathering. Government agencies maintain and analyze citizens&#39; records relating to health, tax records, etc.
+
+### **Industrial use cases of data lakes**
+
+A data lake complements a data warehouse by providing additional [query options](https://docs.microsoft.com/en-us/odata/concepts/queryoptions-overview#). While data warehouses provide organized and structured information, the addition of a data lake helps organizations tap into raw data. Thus, offering more support and insights, considering that data lakes facilitate real-time analytics. Data lakes are applicable in IT, research, and science, among other industries.
+
+- **Research and Science**
+
+The field of science is ever-evolving, and the use of real-time data helps deduct critical insights. Data lakes are applicable for science use because they hold raw data from algorithms and feedback sources in real-time. Science developments rely on the most current and relevant deductions to produce impactful findings and reports. Data lakes help in this aspect because they allow the importation of data that come in real-time.
+
+- **Information technology**
+
+Data lakes enable information technology architects to access data in its most original form. They can [case](https://corrus.com/blog/docs/user-guide/corrus-basics/what-is-case-data/#) this data depending on their needs. Organizations use raw data to create more effective products that meet customers&#39; expectations.
+
+### **Industrial use cases of databases**
+
+Databases are mostly used in data science research and testing. Engineers and data scientists are the primary users of databases.
+
+- **Business data analysis**
+
+Databases have excellent reporting features thus useful for data analysis and trend predictions. For instance, a sales promotion report may show that sales for an item, say A, increased following an effective email promotion. Those of an item, say B, rose following an effective in-store promotion. A productivity report may show that productivity slows drastically on the afternoon of the day before a holiday; the employer might, as well, let the employees go early on the eve of holidays.
+
+- **Telecommunication**
+
+Telecommunication companies use databases to store and generate customer bills, balances for prepaid customers, call logs, among other essential information.
+
+- **Sales**
+
+The sales department benefits significantly from a company&#39;s database. Among other tasks, sales teams use databases to track sales, product performance, and customer information.
+
+### **The future of data lakes, data warehouses, and databases**
+
+The current shift towards cloud-based data platforms to mitigate data issues and manage data suggests that data lakes&#39; future is the cloud. The cloud is scalable and expandable, thus expected to attract more users.
```

Comment:
When you say the future of data lakes is in the cloud, isn't it already the case? I feel you should elaborate on how data lakes are being deployed via the cloud. That would make more sense.